docs/meshstack.billing-configuration.md

```diff
@@ -437,128 +510,42 @@ let example
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
 
-### Configuration
-
-
-<!--snippet:mesh.kraken.productcatalog-->
-
-The following configuration options are available at `mesh.kraken.productcatalog`:
-<!--DOCUSAURUS_CODE_TABS-->
-<!--Dhall Type-->
-```dhall
-{ discounts : List Discount }
-```
-<!--END_DOCUSAURUS_CODE_TABS-->
-
-## Common Chargeable Resources
-
-This section describes the resources that do not strictly belong to a specific cloud platform that can be defined in the catalog.
-Products referring to these resources can be defined for all or multiple cloud platforms.
-
-### meshTenant
-
-Represents a meshTenant. Currently available for Azure, AWS and GCP platforms.
-
-```text
-id: mesh.tenant
-traits:
-  - landingZoneName
-  - platformType
-```
-
-You can define products based on this resource type to charge fees based on the Landing Zone that is in use by a meshTenant
-or to simply charge a fee on the meshTenant itself.
 
-## Chargeback
-
-> Chargeback is the process of allocating IT cost to consumers and feeding it into the company-wide finance and controlling processes.
+## Currency Conversion
 
-Each project in meshStack is associated with a Chargeback Account. meshStack periodically generates [chargeback statements](meshcloud.project-metering.md#chargeback-statements).
+In order to simplify multi-cloud governance, the best practice is to have single chargeback currency. meshStack simplifies the chargeback process with automatic currency conversion capability, namely regardless of the original currency from cloud providers like AWS chargeback statements will be converted to a single currency - Euro. 
 
-The attributes that shall be part of the billing info on the chargeback statements can be configured.
+You have two options based on your [plan](https://www.meshcloud.io/en/pricing/) to enable currency conversion for chargebacks and usage reports:
```

Comment:
d: This link (https://www.meshcloud.io/en/pricing/) just contains a form for a price inquiry, it provides no information about the plan. Not sure what value this link should provide in this context.

Reply:
you are right:)